src/EthMultiVault.sol

```diff
@@ -1988,34 +2177,12 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
     /// @notice The conversion happens in two steps:
     ///  1. First, we get the base assets from the bonding curve:
     ///  $$a_{base} = f_{curve}^{-1}(shares)$$
-    ///  2. Then we apply a pool ratio adjustment to account for divergence between total assets and shares:
-    ///  $$a_{final} = a_{base} \cdot \frac{A_{total}}{S_{total}^{(a)}}$$
-    /// ### Where:
-    ///  1. $A_{total}$ is the total assets in the vault
-    ///  2. $S_{total}^{(a)} = f_{curve}(S_{total})$ is the total shares converted to asset domain
-    ///    using the bonding curve function
-    /// @notice This adjustment is necessary because the actual ratio of shares:assets may drift from
-    /// the bonding curve's 'pure' conversion ratio due to:
-    ///  - Protocol fees being taken in assets
-    ///  - Share burns
-    ///  - Direct asset transfers/airdrops
-    /// @notice The crucial conversion of shares to asset domain ($S_{total}^{(a)}$) ensures we're comparing
-    ///  quantities in the same space, as the bonding curve defines a non-linear relationship
-    ///  between shares and assets.
-    /// @dev TLDR: You need to exchange oranges for apples and they each have their own wholesale discount tiers.
-    ///  So you need to compute their value using a common denominator like dollars.
+    //   2. That's it.  No pool ratio adjustment in this version of the contracts because it only works for
+    //   pro-rata, and that is taken care of in convertToAssets.
     function convertToAssetsCurve(uint256 shares, uint256 id, uint256 curveId) public view returns (uint256) {
         uint256 supply = bondingCurveVaults[id][curveId].totalShares;
         uint256 totalAssets = bondingCurveVaults[id][curveId].totalAssets;
         uint256 assets = _registry().previewRedeem(shares, supply, totalAssets, curveId);
-
```

Comment:
3.2 - Pool Ratio Adjustment Only Works for LinearCurve

We removed the pool ratio adjustment for curve vaults